test/ci/cpu_integration_tests_linux.yml

```diff
@@ -57,6 +57,13 @@ jobs:
       pytest test/integration/test_notebook_linear_explainer.py --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
     displayName: 'Run Linear explainer'
 
+  - bash: |
+      cd /usr/share/miniconda/bin
+      source activate interpret_cpu
+      cd /home/vsts/work/1/s
```

Comment:
we should not need to know so much about the env. Do all tests do the cd cd back?